cache/src/main/java/io/envoyproxy/controlplane/cache/SimpleCache.java

```diff
@@ -196,25 +196,30 @@ public void setSnapshot(T group, Snapshot snapshot) {
       if (status == null) {
         return;
       }
+      for (String typeUrl : Resources.TYPE_URLS) {
```

Comment:
How about a little comment that this `for` makes responses in the right order because `TYPE_URLS` has a list in the right order? It's hard to deduce it from the code.

Reply:
+1